docs/engineering/android-app/developer-setup/testing/readme.mdx

```diff
@@ -0,0 +1 @@
+# Testing
```

Comment:
add generic testing content

Reply:
@Mstjamush I have assigned this PR to you can you please add some notes on integration testing, other quality assurance / forms of testing we do here. Let me know when that's done then I can do a review and merge, thank you

cc @AngelaKabari